### PR TITLE
LibGUI: Fix keyboard shortcut conflicts in TreeView

### PR DESCRIPTION
**Pull request by thankyouverycool**
Changes the shortcut to expand and collapse subtrees from alt to ctrl+right/left arrows. Current shortcut conflicts with navigation controls in file manager and causes erratic behavior.

### CODE REVIEW

File: Libraries/LibGUI/TreeView.cpp [hunk 1]
```diff
@@ -110,13 +110,16 @@ void TreeView::doubleclick_event(MouseEvent& event)
...
 void TreeView::set_open_state_of_all_in_subtree(const ModelIndex& root, bool open)
 {
-    if (root.is_valid())
+    if (root.is_valid()) {
         ensure_metadata_for_index(root).open = open;
+        if (model()->row_count(root)) {
+            if (on_toggle)
+                on_toggle(root, open);
+        }
+    }
```
Comment (awesomekling) [inline Libraries/LibGUI/TreeView.cpp:119]: This is entirely unrelated to the other change, so please put it in a separate commit :)

Comment (thankyouverycool) [reply Libraries/LibGUI/TreeView.cpp:119]: Righto!
